src/test/java/com/wikia/webdriver/common/core/api/EditToken.java

```diff
@@ -78,8 +79,12 @@ public String getEditToken() {
           .disableAutomaticRetries()
           .setSSLHostnameVerifier(NoopHostnameVerifier.INSTANCE)
           .build();
-      HttpGet httpGet = new HttpGet(apiURL);
+      HttpGet httpGet = new HttpGet(UrlBuilder.stripUrlFromEnvSpecificPartAndDowngrade(apiURL));
+      //TODO:Move getBorderProxy() somewhere else
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1135)